### DISCUSSION TITLE
Chapter 2 ~ Items to update

### DISCUSSION THREAD

**Discussion by jehibdon** [Zibiana/QR4SJ#2] [General]
We need to update the introduction to Chapter 2.  This is important as the loan forgiveness material has changed in Summer 2023.

**Comment by jkauba** [comment]
More specifically, after mentioning something that occurred in February, 2023, we say "Finally, on January 10, 2023" and I think this should be in chronological order or include a different transition word such as "however".

**Comment by jehibdon** [comment]
The learning objectives need to be updated to align with the overall learning objectives.

**Comment by MakennaWorley** [comment]
Missing space between _range._What on 2.4, question 6.

2.4 question 1, the question asks for 'public nonprofit', 'private nonprofit', 'private for-porfit' but the excel sheet lists 'Public average', 'Private average', 'Grad prublic avg', and 'Grad private avg'. Does this need to change?

Should the Exercises found at the bottom of 2.7 be moved into their own section or into the "Wrap-Up Exercise"?

-These were found on the old version of the book and not the current one
